test/runtests.jl

```diff
@@ -385,6 +385,20 @@ end
     @test isapprox(coef(gm20)[1:7],
         [2.894527697811509, -0.5693411448715979, 0.08238813087070128, -0.4484636623590206,
          0.08805060372902418, 0.3569553124412582, 0.2921383118842893])
+
+    # Issue #302
+    let df = DataFrame(
```

Comment:
Better use `@testset`, and define `df` inside the block. Also `DataFrame(y=...)` is enough (keep it on a single line).